src/main/java/com/beerjournal/controller/CollectableController.java

```diff
@@ -0,0 +1,65 @@
+package com.beerjournal.controller;
+
+
+import com.beerjournal.datamodel.entity.CollectableObjectEntity;
+import com.beerjournal.datamodel.repository.CollectableObjectsRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.awt.image.BufferedImage;
+import java.io.InputStream;
+import java.util.Collection;
+import java.util.Optional;
+
+@RestController
+@RequestMapping(value = "collectables")
+public class CollectableController {
+
+    @Autowired
+    private CollectableObjectsRepository collectableObjectsRepository;
+
+    @GetMapping("")
+    public Collection<CollectableObjectEntity> getCollectables() {
+        return collectableObjectsRepository.getAll();
+    }
+
+    @GetMapping("{id}")
+    public CollectableObjectEntity getCollectable(@PathVariable("id") String id) {
+        return collectableObjectsRepository.getById(id);
+    }
+
+    @GetMapping("{id}/images")
+    public Optional<BufferedImage> getImage(@PathVariable("id") String id) {
+        return collectableObjectsRepository.getImageForObject(id);
+    }
+
+    @GetMapping("brewery/{name}")
+    public Collection<CollectableObjectEntity> getByBrewery(@PathVariable("name") String name){
+        return collectableObjectsRepository.getByBrewery(name);
+    }
+
+    @DeleteMapping("")
+    public void deleteCollectables() {
```

Comment:
Na delete to akurat faktycznie nie ma wielu opcji, ale czasem byśmy chcieli zwrocic jakieś 404 czy coś.

Ok, to jak tak wolisz to zostaw.

Ewentualnie można później ustawic jakiś handler na wyjatki żeby zwracac jakieś statusy w razie bledu